<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringTableGeneratorDialog</class>
 <widget class="QDialog" name="StringTableGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>String Generator</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPlainTextEdit" name="input"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="text">
        <string>Copy</string>
       </property>
       <property name="icon">
        <iconset theme="edit-copy"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset theme="edit-clear"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="output">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
